pkgs/top-level/haxe-packages.nix

```diff
@@ -28,12 +28,13 @@ let
       ...
     } @ attrs:
       stdenv.mkDerivation (attrs // {
-        name = "${libname}-${version}";
+        pname = libname;
+        inherit version;
 
         buildInputs = (attrs.buildInputs or []) ++ [ haxe neko ]; # for setup-hook.sh to work
         src = fetchzip rec {
           name = "${libname}-${version}";
-          url = "http://lib.haxe.org/files/3.0/${withCommas name}.zip";
+          url = "https://lib.haxe.org/files/3.0/${withCommas name}.zip";
```

Comment:
```suggestion
          url = "https://lib.haxe.org/files/${lib.versions.majorMinor version}/${withCommas name}.zip";
```

Reply:
it is not `majorMinor` of some version we have here.
It is the version of repository web api